target/linux/ramips/image/Makefile

```diff
@@ -111,6 +111,12 @@ define Build/elx-header
 	rm -rf $(KDIR)/tmp/$(DEVICE_NAME).header
 endef
 
+define Build/netis-tail
+	echo -n $(1) >> $@
+	echo -n $(UIMAGE_NAME)-yun | $(STAGING_DIR_HOST)/bin/mkhash md5 | \
+		sed 's/../\\\\x&/g' | xargs echo -ne >> $@
+endef
```

Comment:
Please highlight something like this more aggressively next time. I'd have merged this here directly then ... :-)

Reply:
Okay, noted.